library/testing/PseudoUnitTest.h

```diff
@@ -1,6 +1,7 @@
 #ifndef PseudoUnitTest_h
 #define PseudoUnitTest_h
 
+#include "types.h"
```

Comment:
not sure why this would be needed

Reply:
I would assume it's to know about `std::ostream& operator<<(std::ostream& os, const f3d::vector3_t& vec)`.

I think it would be better if it was not included from inside `PseudoUnitTest.h` (to keep that bit of code as "standalone" as possible) but I don't know if it's doable. Would it work to have `#include "types.h"` before `#include "PseudoUnitTest.h"` in `TestSDKOptionsIO.cxx`?